src/core/api/heap_api.cpp

```diff
@@ -63,6 +63,17 @@ void otHeapFree(void *aPointer)
 
 #else // OPENTHREAD_RADIO
 
+#if OPENTHREAD_CONFIG_EXTERNAL_HEAP_ENABLE
+void *otHeapCAlloc(size_t aCount, size_t aSize)
+{
+    return ot::Instance::HeapCAlloc(aCount, aSize);
+}
+
+void otHeapFree(void *aPointer)
+{
+    ot::Instance::HeapFree(aPointer);
+}
+#else
```

Comment:
```suggestion
```

can we use the same implementation for both case?

Reply:
good suggestion.